src/middle-end/auth/index.js

```diff
@@ -99,17 +99,17 @@ module.exports = (m) => {
                 }
             }),
             forgotPassword: createAction(FORGOT_PASSWORD, {
-                handler: async ({ email }) => {
+                handler: async ({ username }) => {
 
                     const client = getClient();
-                    await client.post('/forgot-password', { username: email });
+                    await client.post('/forgotpassword', { username });
                 }
             }),
             resetPassword: createAction(RESET_PASSWORD, {
-                handler: async ({ token, hash, password }) => {
+                handler: async (args) => {
 
                     const client = getClient();
-                    await client.post('/reset-password', { token, hash, password });
+                    await client.post('/resetpassword2', args);
```

Comment:
Keeping this general (`args`) because when this route can _also_ accept `oldPassword` alongside `newPassword` so the user can reset their password while authenticated (without sending a "forgot password" email). We're not currently handling this on the front end, but we _could_.

Reply:
```suggestion
                    await client.post('/reset-password', args);
```

I'm told this is `/reset-password` again as of `pallies@0.7.2`

Reply:
I could also see (and kinda lean toward) slicing this as following pallies and documenting all possible args here, leaving it opaque how they'll be used under the hood, such that we at least know what you could possibly pass here.

Worth testing that passing along undefined params in this way actually works / doesn't lead to any issues on the pallies side, of course